app/src/organisms/CommandText/PipettingCommandText.tsx

```diff
@@ -1,10 +1,6 @@
 import { useTranslation } from 'react-i18next'
 
-import {
-  CompletedProtocolAnalysis,
-  getLabwareDefURI,
-  RobotType,
-} from '@opentrons/shared-data'
+import { getLabwareDefURI, RobotType } from '@opentrons/shared-data'
```

Comment:
nit, RobotType is a type and can be separated 
```suggestion
import { getLabwareDefURI} from '@opentrons/shared-data'
import type {RobotType} from '@opentrons/shared-data'

```